src/recipes/eas.ts

```diff
@@ -0,0 +1,150 @@
+import { REPOSITORY_SECRETS_HELP_URL } from '../constants'
+import { CycliRecipe, CycliToolbox, ProjectContext } from '../types'
+import { join } from 'path'
+import { recursiveAssign } from '../utils/recursiveAssign'
+import { addTerminatingNewline } from '../utils/addTerminatingNewline'
+
+const FLAG = 'eas'
+
+const patchEasJson = async (
+  toolbox: CycliToolbox,
+  withIOSCredentials: boolean
+): Promise<void> => {
+  const patch = {
+    build: {
+      development: {
+        developmentClient: true,
+        distribution: 'internal',
+        android: {
+          buildType: 'apk',
+        },
+        channel: 'development',
+      },
+    },
+  }
+
+  if (!withIOSCredentials) {
+    patch.build.development['ios'] = {
+      simulator: true,
+    }
+  }
+
+  await toolbox.patching.update('eas.json', (config) => {
```

Comment:
```suggestion
  await toolbox.patching.update('eas.json', (config) => recursiveAssign(config, patch))
```

Reply:
nice, changed in d2aad9439aab993dceaa7880f622a4f7a3f1b78e